package.json

```diff
@@ -46,11 +48,13 @@
     "styled-jsx": "3.2.5"
   },
   "scripts": {
+    "clean": "npx rimraf build/ dist/",
     "prestart": "yarn localize && d2-manifest package.json manifest.webapp",
     "start": "react-scripts start",
     "prebuild": "yarn test",
     "build": "rm -rf build/ && yarn localize && d2-manifest package.json manifest.webapp && react-scripts build && yarn run manifest && cp -r i18n icon.png build",
     "build-webapp": "yarn build && rm -f $npm_package_name.zip && cd build && zip -r ../$npm_package_name.zip *",
+    "build-scheduler": "npx ncc build src/scheduler/cli.ts -m",
```

Comment:
[very minor] Those two npx in clean/build-scheduler are necessary? It should use the ones in node_modules/.bin by default.

Reply:
That's true, I copy/pasted my commands from the terminal